cellengine/resources/scaleset.py

```diff
@@ -72,11 +72,11 @@ def apply(
         """Apply the scaleset to a file.
 
         Args:
-            file (_id or FcsFile): The file to which this scaleset will be
+            file: The file to which this scaleset will be
                 applied.
-            clamp_q (bool): Clamp the output to the scale's minimum and maximum
+            clamp_q: Clamp the output to the scale's minimum and maximum
                 values.
-            in_place (bool): If True, updates the FcsFile.events; if
+            in_place: If True, updates the FcsFile.events; if
```

Comment:
https://github.com/primitybio/cellengine-python-toolkit/pull/136#discussion_r796217240 - I thought we're supposed to repeat types in both the function signature and in the doc string?

Reply:
It appears that `griffe` (new version of mkdocstrings) is quite happy without; I _think_ the reason I was leaving them in was in cases where it was more informative to overwrite the function signature

Reply:
Okay, so griffe caught that `_id` is not a type (should have been string), and instead of fixing line 75, it can just be removed, got it.